code/_compile_options.dm

```diff
@@ -32,6 +32,13 @@
 
 #endif //ifdef REFERENCE_TRACKING
 
+// If this is uncommented, we do a single run though of the game setup and tear down process with unit tests in between
+// #define UNIT_TESTS
```

Comment:
duplicate define located in l. 53. Remove the old one if you want to move it here.